templates/terraform/examples/base_configs/iam_test_file.go.erb

```diff
@@ -174,7 +179,10 @@ func testAcc<%= resource_name -%>IamBinding_updateGenerated(context map[string]i
 <%= example.config_test_body -%>
 
 resource "<%= terraform_name -%>_iam_binding" "foo" {
-	<%= object.name.underscore -%> = "${<%= terraform_name -%>.<%= example.primary_resource_id -%>.id}"
+	<%= params.last.underscore -%> = "${<%= parent_resource_type_type -%>.<%= example.primary_resource_id -%>.<%= object.iam_policy.parent_resource_attribute -%>}"
```

Comment:
This currently uses resource.id as IAM resource *should* be able to parse their parent's id. I plan to change this to use the same code as the documentation example generation, which sets each field in the URL based on the fields of the parent resource.

For example: pubsub topic would set `topic = google_pubsub_topic.my_topic.name` and `project = google_pubsub_topic.my_topic.project`.